Author: Scot Doyle <lkml14@scotdoyle.com>
Description: load_linux: correct a type
 Correct base's type to match its initialization from prot_mode_base and
 passage to syslinux_memmap_find(). Tested with extlinux.

diff -Naurp syslinux.orig/com32/lib/syslinux/load_linux.c syslinux/com32/lib/syslinux/load_linux.c
--- syslinux.orig/com32/lib/syslinux/load_linux.c
+++ syslinux/com32/lib/syslinux/load_linux.c
@@ -155,8 +155,8 @@ int bios_boot_linux(void *kernel_buf, si
 		    char *cmdline)
 {
     struct linux_header hdr, *whdr;
-    size_t real_mode_size, prot_mode_size, base;
-    addr_t real_mode_base, prot_mode_base, prot_mode_max;
+    size_t real_mode_size, prot_mode_size;
+    addr_t real_mode_base, prot_mode_base, prot_mode_max, base;
     addr_t irf_size;
     size_t cmdline_size, cmdline_offset;
     struct setup_data *sdp;
